File: Python/UD 0/Boletin 0/Ejercicio10.py
numero1 = int(input("Dame un número: "))
numero2 = int(input("Dame otro número: "))

if numero2 != 0:
    division = numero1 / numero2
    print("Resultado:", division)
else:
    print("No se puede dividir entre 0, inténtalo de nuevo")
    numero2 = int(input("Dame otro número: "))

    if numero2 != 0:
        division = numero1 / numero2
        print("Resultado:", division)
    else:
        print("Sigue siendo 0. No se puede dividir.")

File: Java/Unidad1/src/main/java/Unidad2/BoletinIF/Ejercicio10.java
package Unidad2.BoletinIF;
import java.util.Scanner;

public class Ejercicio10 {
    static void main(String[] args) {

        Scanner escaner = new Scanner(System.in);

        System.out.print("Dame un numero: ");
        int numero1 = escaner.nextInt();

        System.out.print("Dame otro numero: ");
        int numero2 = escaner.nextInt();

        float resultado = (float) numero1 / numero2;

        if (numero2 != 0) {
            System.out.println("El resultado de la division es: " + resultado);
        }
        else {
            System.out.println("El segundo numero no puede ser cero");

        }
    }
}
